crates/ln-dlc-node/src/node/invoice.rs

```diff
@@ -152,7 +164,7 @@ where
         tracing::info!(peer_id=%target_node, %intercept_scid, "Successfully created intercept scid for payment routing");
         InterceptableScidDetails {
             scid: intercept_scid,
-            jit_routing_fee_millionth: jit_fee_rate_basis_point * 100,
+            jit_routing_fee_millionth: jit_routing_fee_millionth * 100,
```

Comment:
🤔 I would remove the `jit` prefix from that variable. As the `just in time` aspect on how the channel got created has little to do on how much we charge on routing fees.

Note, we only need to provide that information here, because 1) the channel might not exist yet and 2) private channels don't get announced, and the sender does not know the required fees for constructing a payment to that channel. The later scenario should be the most common though.

Reply:
I think the last patch I am gonna add to this PR will address this.

> Note, we only need to provide that information here, because 1) the channel might not exist yet and 2) private channels don't get announced, and the sender does not know the required fees for constructing a payment to that channel. The later scenario should be the most common though.

I guess this comment reflects the fact we are using this API for multiple purposes. If we route a "regular" payment to the app what you say makes sense. It might mean that we always want/need to add this route hint to the invoice even if we don't want to intercept it!

For the purpose that it was designed for, this information is simply necessary because we _need_ the last `RouteHintHop` to be able to intercept the channel.

Reply:
It was actually merged onto https://github.com/get10101/10101/pull/951/commits/2ddbee01ccb53fe135f3af5497ef239f8693c4b8 in the end, since the test weren't passing anyway.